lib/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+package constants
+
+const (
```

Comment:
None of these are consts. There are default values, but the serving values are actually defined by the configuration.

Reply:
How about I prepend these with `Default`?